magit-annex.el

```diff
@@ -679,8 +680,11 @@ on the file at point.
         (insert (format "%s %s" locs file))
         (forward-line)))))
 
-(defun magit-annex-list-file-at-point ()
-  (cdr (magit-section-when annex-list-file)))
+(defun magit-annex-list-files-at-point-or-region ()
```

Comment:
Hmm, this is only called in one place.  I'm not quite sure why I felt this deserved its own function.  I'd be in favor of moving it inline.  Could you add a commit before this "region files" one that inlines magit-annex-list-file-at-point's body?